config/configload/merge.go

```diff
@@ -57,6 +62,52 @@ func absBackendBlock(backendBlock *hclsyntax.Block) {
 	}
 }
 
+func absInBackends(block *hclsyntax.Block) error {
+	for _, subBlock := range block.Body.Blocks {
+		if subBlock.Type == errorHandler {
+			return absInBackends(subBlock)
+		}
+
+		if subBlock.Type == proxy || subBlock.Type == request {
+			var backends int
+
+			for _, subSubBlock := range subBlock.Body.Blocks {
+				if subSubBlock.Type == backend {
+					absBackendBlock(subSubBlock) // Backend block inside a proxy or request block
+
+					backends++
+				}
+			}
+
+			if _, ok := subBlock.Body.Attributes[backend]; ok {
+				backends++
+			}
+
+			if backends > 1 {
+				return createError(errMultipleBackends, block)
+			}
+		}
+	}
+
+	return nil
+}
+
+// newErrorHandlerKey returns a merge key based on a possible mixed error-kind format.
+// "label1" and/or "label2 label3" results in "label1 label2 label3".
+func newErrorHandlerKey(block *hclsyntax.Block) (key string) {
```

Comment:
Reason for moving this method to top of the file? Helper methods should be not at first place due to readability reasons.

Reply:
All helper methods are currently placed in the top of this file. We can move all helper methods to the bottom of file.